### PR TITLE
[otbn] Add a note to otbn/README.md explaining insns.yml

### PR DESCRIPTION
**Pull request by rswarbrick**

### CODE REVIEW

File: hw/ip/otbn/README.md [hunk 1]
```diff
@@ -22,3 +22,12 @@ compiled as follows.
 ```sh
 fusesoc --cores-root=. run --target=sim --setup --build lowrisc:systems:top_earlgrey_verilator
 ```
+
+## Working with the ISA
```
Comment (imphil) [inline hw/ip/otbn/README.md:26]: Can you make this `###` and the one above as well? (I did miss that in the previous PR unfortunately.)

File: hw/ip/otbn/README.md [hunk 2]
```diff
@@ -22,3 +22,12 @@ compiled as follows.
...
+
+The instruction set is described in machine readable form in `data/insns.yml`.
+This is parsed by Python code in `util/insn_yaml.py`, which runs various sanity
+checks on the data. Current tooling that uses this information:
+
+  - `util/yaml_to_doc.py`: Generates a Markdown snippet which is included in
+    the main OpenTitan help text.
```
Comment (imphil) [inline hw/ip/otbn/README.md:33]:
```suggestion
    the OTBN specification.
```

File: hw/ip/otbn/README.md (hunk 1 above)
Comment (rswarbrick) [reply hw/ip/otbn/README.md:26]: Yep, can do!

File: hw/ip/otbn/README.md (hunk 2 above)
Comment (rswarbrick) [reply hw/ip/otbn/README.md:33]: That's much better, thanks.

**Pull request merged**